DirectEmployers/MyJobs
# [PD 3597] page numbers do not reset

## What's this PR do?
PR resets the active page state when a new dimension is clicked on for the table pagination

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside of the analytics platform

## How should this be manually tested?
Pull down the repository. Go into analytics and go to any page on the table. Once there, click on a new dimension and you will see the pagination state updates to reflect the first page and not the previous page you were on with the other dimension

## Any background context you want to provide?
